.ci/pipeline/ci.yml

```diff
@@ -79,21 +77,16 @@ jobs:
       Python3.11_Sklearn1.2:
         PYTHON_VERSION: '3.11'
         SKLEARN_VERSION: '1.2'
+      Python3.11_SklearnMain:
```

Comment:
Isn't this overkill for CI? might be worth to keep it just in Nightly?

Reply:
This step is in PR scope now to check if it works. But I think it is not real overkill because this step takes ~10 minutes and ~25% CI time resources additionally.

Reply:
Yes. but there might be problem with scikti that would be blocking PRs. 
so in case of nightly runs we would have daily validation but failures against scikit master wouldn't be blocking PRs